README.md

```diff
@@ -165,6 +165,13 @@ Starting holorepository-core_holorepository-ui-server_1                      ...
 Creating holorepository-core_holostorage-accessor_1                          ... done
 ```
 
+In cases where only selective services are required, they can be started by specifying the service name
```

Comment:
This seems a bit redundant with line 177 - 180 now, don't you think?

Reply:
oh yes true, I missed that part.